modules/multer.js

```diff
@@ -12,4 +12,7 @@ const upload = multer({
     storage: storage
 }).single('scanmeds')
 
-module.exports = upload
\ No newline at end of file
+module.exports = upload
+
+// const storage = multer.memoryStorage()
```

Comment:
are you going to use this commented section?

Reply:
Still contemplating, but will remove it for now. Was thinking about a way to not save images, privacy wise.

Reply:
ah yea I thought so, you can keep it there for later use that's fine 👯